utils/github/parser.py

```diff
@@ -37,8 +41,20 @@ def _parse(self, text):
             if stripped == 'I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en':
                 self.legal = True
 
-            if stripped == 'Category (leave one):':
+            category_headers = (
+                'Category (leave one):',
+                'Changelog category (leave one):',
+                'Changelog category:',
+                'Category:'
+            )
+
+            if stripped in category_headers:
                 next_category = True
 
         if category in Description.MAP_CATEGORY_TO_LABEL:
             self.label_name = Description.MAP_CATEGORY_TO_LABEL[category]
+        else:
+            if not category:
+                print('Cannot find category in pr description')
```

Comment:
Maybe print PR number too?

Reply:
It is printed in message before. like
`Trying to auto-label pull-request: 8882`